commands/permissions.go

```diff
@@ -132,3 +177,54 @@ Scheme Managed: no
 
 	return nil
 }
+
+func assignUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	role, response := c.GetRoleByName(args[0])
+	if response.Error != nil {
+		return response.Error
+	}
+
+	users := getUsersFromUserArgs(c, args[1:])
+
+	for _, user := range users {
```

Comment:
we should check that all users have been found when iterating, maybe adding something like:

```suggestion
	for i, user := range users {
		if user == nil {
			printer.PrintError("Couldn't find user '" + args[i+1] + "'")
			continue
		}
```

Reply:
I've update it, thanks.